<?xml version="1.0"?>
<launch>
  <node name="squirrel_safety_node" pkg="squirrel_safety" type="squirrel_safety_node.py" output="screen" respawn="true">
    <param name="airskin_topic" value="/airskin/arm_bumper" type="string"/>
    <param name="bumper_topic" value="/bumper" type="string"/>
    <param name="wrist_topic" value="/wrist/wrist_bumper" type="string"/>
  </node>)
</launch>
